doc/user/load_save.md

```diff
@@ -604,11 +607,100 @@ mlpack::data::ResizeImages(images, info, 160, 160);
 // contains the names of the resized images.
 std::vector<std::string> smSheeps =
     {"sm_sheep_1.jpg", "sm_sheep_2.jpg", "sm_sheep_3.jpg", "sm_sheep_4.jpg",
-     "sm_sheep_5.jpg", "sm_sheep_6.jpg"};
+     "sm_sheep_5.jpg", "sm_sheep_6.jpg", "re_sheep_7.jpg", "re_sheep_8.jpg",
+     "re_sheep_9.jpg"};
 
 mlpack::data::Save(smSheeps, images, info, false);
 ```
 
+### Crop and Resize images.
+
+In addition to resize images, mlpack provides also crop and resize
+functionality. The objective of this feature is to keep the aspect ratio in
+images where dimension differs largely. This idea is simple, the bigger
+dimension is found and then the images are Resized and cropped on that
+dimension only. This algorithm is suitable if the area of interest is in the
+center / or surrounding the center of the image, allowing to keep a high
+resolution of the area of interest when the image is resized. In the following
+example, we show how an image of a cat would look like after cropping or
+resizing:
+
+Original image:
+
+<p align="center" width="100%">
+    <img width="100%" src="../img/cat.jpg">
+</p>
+
+Resized image:
+
+<p align="center" width="100%">
+    <img width="100%" src="../img/resized_cat.jpg">
+</p>
+
+Cropped image:
+
+<p align="center" width="100%">
+    <img width="100%" src="../img/cropped_cat.jpg">
```

Comment:
The image you added is called `croped_cat.jpg`, so I think you need to move it to `cropped_cat.jpg` :)